### ISSUE TITLE
Add sign in & sign up Page

### ISSUE THREAD

**Issue by kajal-alpha** [techcombd/asset-manager#25]
## I'm submitting a...

<pre><code>
[ ] Bug report  
[ ] Performance issue
[x] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Current behavior

There is no login / Sign up Page. Directly showing the dashboard.

## Expected behavior

There should be showing login page first. user can view the dashboard after logging in.

**Comment by mahedee** [comment]
We can solve this issue later. Is there any dependency on this issue?

**Comment by ratanparai** [comment]
I agree with @mahedee, we can add authentication and authorization later after we have something to protect 😁
